identity/docker-compose-identity.yaml

```diff
@@ -0,0 +1,10 @@
+services:
```

Comment:
It looks like this file is only for testing, but otherwise I can't see any instructions on how to use it. Should we remove it here and rather add it as part of automatic test coverage?

Reply:
This file should be removed, right?